tests/testthat/test_calcWbgt.R

```diff
@@ -0,0 +1,53 @@
+test_that("calculate calcWbgt with parameters", {
+  expect_equal(
+    isTRUE(
+      all.equal(
+        calcWbgt(twb = 25, tg = 32, tdb = 20, with_solar_load = TRUE),
```

Comment:
these values should not be hardcoded but as we previously discussed should be taken from the external validation table. Please change them accordingly